Containerfile

```diff
@@ -0,0 +1,3 @@
+FROM fedora-minimal:38
+
+RUN dnf5 install -y poetry git
```

Comment:
If we're going to put poetry in a container I'd prefer we use latest upstream vs distro version

Reply:
This is a good point. I realized even F38 doesn't have poetry 2+ in repo, so I can't use it for my projects the way I mentioned above.

Of course, this is sending us down the path of creating a generic poetry image. :-) 

If there's a generic poetry image done separate from this, then this PR would change to "install poetry (see website for instructions) or just use this shell alias to run a docker container.

Reply:
Not against that but there's no official image: https://github.com/python-poetry/poetry/issues/4036

I prefer to get my tooling from fedora for the most part and didn't really look at what upstream recommends from installing, if you know how to do that I can amend this PR

The script is really what makes things easy, the container itself isn't too relevant imo

Reply:
I'm actively working on something to contribute here... not quite ready yet.